internal/server/proto/server.proto

```diff
@@ -3971,6 +3974,17 @@ message Token {
     string deployment_id = 1;
   }
 
+  // The Trigger message is a kind of token type that is only used for
+  // authenticated trigger URL requests. It should not have any other
+  // authorized access to make requests in any other API endpoint.
+  message Trigger {
+    // The user that initiated the trigger token generation
+    string from_user_id = 1;
+
+    // The login token we'd like to create.
+    Login login = 2;
```

Comment:
I don't think you need any fields in the `Trigger` message at all

Reply:
Ah yeah, I don't think we really need the login message. The user that created the token might be useful though?

Reply:
yeah that sounds good!